src/main/java/org/apache/ibatis/jdbc/SqlRunner.java

```diff
@@ -220,44 +199,37 @@ private void setParameters(PreparedStatement ps, Object... args) throws SQLExcep
     }
   }
 
+  /**
+   *  ResultSet should't was closed in here,it should be closed in caller
+   */
```

Comment:
You wrote...

> I found SqlRunner#insert invoked SqlRunner#getResults and the ResultSet has been closed in SqlRunner#insert .
> SqlRunner#selectAll has the same situation , too. So, in my opinion, ResultSet should be closed in caller.

, but I don't see any code closing `ResultSet` in both methods.

- insert()
https://github.com/mybatis/mybatis-3/blob/d89b30095f4687ae96119d70ef4622128efdd73f/src/main/java/org/apache/ibatis/jdbc/SqlRunner.java#L114

- selectAll()
https://github.com/mybatis/mybatis-3/blob/d89b30095f4687ae96119d70ef4622128efdd73f/src/main/java/org/apache/ibatis/jdbc/SqlRunner.java#L83-L84

Am I missing something?

Reply:
You're right. i miss something .Already fixed in new commit.thinks for you code review